ush/setup.sh

```diff
@@ -1123,51 +1127,52 @@ files are located.  Currently, this is not the case:
   FIXLAM_NCO_BASEDIR = \"${FIXLAM_NCO_BASEDIR}\"
 Resetting GWD_HRRRsuite_BASEDIR to FIXLAM_NCO_BASEDIR.  Reset value is:
   GWD_HRRRsuite_BASEDIR = \"${GWD_HRRRsuite_BASEDIR}\""
+      fi
     fi
-  fi
 #
 # Check that GWD_HRRRsuite_BASEDIR exists and is a directory.
 #
-  if [ ! -d "${GWD_HRRRsuite_BASEDIR}" ]; then
-    print_err_msg_exit "\
+    if [ ! -d "${GWD_HRRRsuite_BASEDIR}" ]; then
```

Comment:
There are some differences starting here that I think are just formatting differences (from about line 1135 to 1173 in the updated file in a few different blocks).  Could these be cleaned up to make the actual changes clearer?

Reply:
Yeah, those are just formatting differences.  The actual change is simply to add the if [ "PREDEF_GRID_NAME" != "RRFS_FIREWX_1.5km" ] statement at the beginning.  I think what happened is I did not indent all of the commented lines, which is why the changes look so messy.  I can clean that up.

Reply:
I indented the lines that I did not indent previously, but it's still a little tough to discern the actual changes from formatting differences.  Let me know if I should do anything more.  Thanks

Reply:
I think you modified more than was necessary - seems like there are more differences than before, but hard to tell now.    Could you try not indenting the comment blocks starting with "Ensure that" at about lines 1144 and 1157?

Reply:
Okay I went ahead and remove the unnecessary indenting and now the changes should be much clearer.  I am used to working with Python code where indenting is important, but in this case it added complexity where it wasn't needed.